gadget/gadget.go

```diff
@@ -328,7 +330,7 @@ func ReadInfo(gadgetSnapRootDir string, classic bool) (*Info, error) {
 		switch v.Bootloader {
 		case "":
 			// pass
-		case "grub", "u-boot", "android-boot":
+		case "grub", "u-boot", "android-boot", "lk":
 			bootloadersFound += 1
 		default:
 			return nil, errors.New("bootloader must be one of grub, u-boot or android-boot")
```

Comment:
this error message needs fixing

Reply:
what happened to error starting with 'cannot ....' here :-P
Error updated :)